examples/cookbook/package.json

```diff
@@ -11,6 +11,6 @@
   "dependencies": {
     "chalk": "^4.1.1",
     "homedir": "^0.6.0",
-    "near-api-js": "file:../../lib/index.js"
+    "near-api-js": "file:../../"
```

Comment:
Just curious why we had to change this here?

Reply:
Local `naj` files were not used. `yarn` was giving me this error:
```txt
error An unexpected error occurred: "ENOTDIR: not a directory, scandir '/home/serhii/.cache/yarn/v6/npm-near-api-js-0.0.0-6b3d283e-f71e-46ce-84aa-ea1ce6432df4-1637087145174/node_modules/near-api-js'".
```
And this format is encouraged in [npm docs](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies).